pages/categories/index.js

```diff
@@ -3,7 +3,9 @@ import SubCatCard from "~/components/SubCatCard";
 import getColorForCategoryId from "~/lib/getColorForCategoryId";
 import convertSlugToTitle from "~/lib/convertSlugToTitle";
 
-const Categories = () => {
+const Categories = ({ issuesCategories }) => {
```

Comment:
The build is failing because of this Negin. You have the variable issuesCategories as a prop as well as renaming `data` from the `useSWR` call on line 10 to issuesCategories as well.

This is a problem, but the bigger question is which one should you be using? We want to maintain a single source of truth so I'd expect you to keep _either_ the prop _or_ the SWR call on line 10.

Reply:
I managed to make 100s of problems trying to fix the conflicts 😖 , I think this was one of them :D I think my latest commit should finally fix it 🙏🏼